tests/App/config/services.php

```diff
@@ -23,5 +24,7 @@
         ->set(DynamicController::class)
             ->public()
             ->tag('container.service_subscriber')
-            ->call('setContainer', [new ReferenceConfigurator(ContainerInterface::class)]);
+            ->call('setContainer', [new ReferenceConfigurator(ContainerInterface::class)])
+
+        ->set('logger', NullLogger::class);
```

Comment:
with this we reduce the output between tests. I guess it could be applied to all repositories that use kernelTestCase.